system/logaccess/Azure/LogAnalytics/CurlClient/AzureLogAnalyticsCurlClient.cpp

```diff
@@ -395,6 +395,13 @@ AzureLogAnalyticsCurlClient::AzureLogAnalyticsCurlClient(IPropertyTree & logAcce
         }
         else if (streq(logMapType, "host") || streq(logMapType, "node"))
         {
+            if (streq(logMapType, "host"))
+            {
+                OWARNLOG("%s: 'host' LogMap entry is ambigious and should be avoided!", COMPONENT_NAME);
```

Comment:
should this say that it is deprecated and should be replaced by "node"?

Reply:
> should this say that it is deprecated and should be replaced by "node"?

I considered using that term, but ultimately felt it wasn't accurate since it's allowed by the schema, and utilized by the clients and will likely not be removed in the future. All that said, I can change if we feel strong about that wording.